.buildkite/pipeline.yml

```diff
@@ -2,7 +2,7 @@
 common_params:
   # Common plugin settings to use with the `plugins` key.
   - &common_plugins
-    - automattic/a8c-ci-toolkit#3.0.0
+    - automattic/a8c-ci-toolkit#3.0.1
     - automattic/git-s3-cache#1.1.4:
```

Comment:
To keep in mind of the future, @jkmassel mentioned that once we move to the Apple Silicon CI stack, we will no longer need to manage Git caches via this plugin.

Reply:
Yep, I am working on a WCiOS branch to move over to Apple Silicon CI that will remove the `git-s3-cache` plugin from here 🙂